python/lsst/ts/observatory/control/mock/atcs_mock.py

```diff
@@ -210,6 +223,31 @@ async def atptg_telemetry(self):
 
             await asyncio.sleep(1.0)
 
+    async def ataos_telemetry(self):
+
+        self.ataos.evt_correctionEnabled.set_put()
+
+        correction_times = cycle((1, 1, 2, 4, 10))
+        while self.run_telemetry_loop:
+            if self.ataos.evt_summaryState.data.summaryState == salobj.State.ENABLED:
```

Comment:
Not entirely sure, but shouldn't this `if` be outside the `while` loop?

Reply:
no, this loop is constantly running, even of the state is not enabled.

Reply:
Yeah I figured which is why I wasn't sure. Thanks for the explanation.